jsdoc-typeof-plugin.js

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Hmm, maybe this file should be placed somewhere else... @gkatsev any thoughts?

Reply:
in build?

Reply:
Thx. It sounds good! I have just updated the PR.